app/controllers/categories_controller.rb

```diff
@@ -28,6 +28,22 @@ def destroy
     end
   end
 
+  def index
+    @categories = Category.all
+    render json: @categories
+  end
+
+  def get_resources_in
+    resources = []
+    @category = Category.find(params[:id])
+    @category.partnering_organizations.each do |partner_org| 
+      resources += partner_org.resources
+    end
+    render json: resources, each_serializer: ResourceSerializer, scope: {
+      current_veteran: current_veteran
```

Comment:
I don't think this needs to be here if we're not using `current_veteran` in the serializer.

Although, I'm a little confused why you got rid of that upvote thing in the serializer?